Android Builds failing on latest flutter Stable channel with v2.3.0

```
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.2.1/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (17, 7): Redeclaration: IntercomFlutterPlugin
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (20, 7): Redeclaration: IntercomFlutterPlugin
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (28, 58): No value passed for parameter 'application'
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (30, 65): No value passed for parameter 'application'
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (37, 36): Unresolved reference: NonNull
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (39, 56): No value passed for parameter 'application'
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (41, 63): No value passed for parameter 'application'
e: .pub-cache/hosted/pub.dartlang.org/intercom_flutter-2.3.0/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt: (207, 38): Unresolved reference: NonNull

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':intercom_flutter:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 29s
Exception: Gradle task assembleDebug failed with exit code 1
Exited (sigterm)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 2.3.1
+* Fix Android build issue
+* Updated Android dependencies
+
 ## 2.3.0
 * Migrate Android side to Flutter's v2 Android Plugin APIs
 

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -2,14 +2,14 @@ group 'io.maido.intercom'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = '1.4.10'
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:4.1.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
@@ -25,7 +25,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'
@@ -41,5 +41,5 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    implementation 'io.intercom.android:intercom-sdk:7.2.+'
+    implementation 'io.intercom.android:intercom-sdk:8.2.+'
 }

--- a/android/gradle/wrapper/gradle-wrapper.properties
+++ b/android/gradle/wrapper/gradle-wrapper.properties
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip

--- a/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt
+++ b/android/src/main/kotlin/io/maido/intercom/IntercomFlutterPlugin.kt
@@ -1,6 +1,7 @@
 package io.maido.intercom
 
 import android.app.Application
+import androidx.annotation.NonNull
 import io.flutter.embedding.engine.plugins.FlutterPlugin
 import io.flutter.plugin.common.EventChannel
 import io.flutter.plugin.common.MethodChannel

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,7 +1,7 @@
 name: intercom_flutter
 description: Flutter plugin for Intercom integration. Provides in-app messaging
   and help-center Intercom services
-version: 2.3.0
+version: 2.3.1
 homepage: https://github.com/v3rm0n/intercom_flutter
 
 dependencies:

